File: FrontEnd/src/app/features/reservations/manifest/classes/view-models/list/manifest-passenger-vm.ts
import { ManifestNationalityVM } from './manifest-nationality-vm'
import { ManifestPortVM } from './manifest-port-vm'
import { SimpleEntity } from 'src/app/shared/classes/simple-entity'

export interface ManifestPassengerVM {

    id: number
    lastname: string
    firstname: string
    birthdate: string
    passportNo: string
    passportExpiryDate: string
    phones: string
    remarks: string
    specialCare: string
    gender: SimpleEntity
    nationality: ManifestNationalityVM
    port: ManifestPortVM

}

File: API/Features/Reservations/Manifest/ViewModels/List/ManifestPassengerVM.cs
using API.Infrastructure.Classes;

namespace API.Features.Reservations.Manifest {

    public class ManifestPassengerVM {

        public string Lastname { get; set; }
        public string Firstname { get; set; }
        public SimpleEntity Gender { get; set; }
        public ManifestNationalityVM Nationality { get; set; }
        public string Birthdate { get; set; }
        public string PassportNo { get; set; }
        public string PassportExpiryDate { get; set; }
        public string Phones { get; set; }
        public string Remarks { get; set; }
        public string SpecialCare { get; set; }
        public ManifestPortVM Port { get; set; }

    }

}